pages/_app.tsx

```diff
@@ -0,0 +1,123 @@
+import { AppProps } from 'next/app'
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import * as Fathom from 'fathom-client'
+import { ChakraProvider } from '@chakra-ui/react'
+import { NextSeo, SocialProfileJsonLd } from 'next-seo'
+import Head from 'next/head'
+
+// import 'cal-sans'
+
+import theme from '../src/theme/theme'
+import Layout from '../src/components/layout/layout'
+import CmdPalette from '../src/components/cmd-palete/command-index'
+import CmdPaletteProvider from 'src/providers/cmd-palette-provider'
+// import '../styles/prism.css'
+import '@code-hike/mdx/dist/index.css'
+
+// import '/Users/tre/tregalloway.com/styles/style.css'
+
+const App = ({ Component, pageProps }: AppProps) => {
+    const router = useRouter()
+
+    useEffect(() => {
+        // Initialize Fathom when the app loads
+        // Example: yourdomain.com
+        //  - Do not include https://
+        //  - This must be an exact match of your domain.
+        //  - If you're using www. for your domain, make sure you include that here.
+        Fathom.load('UPJAYRLL', {
+            includedDomains: ['tregalloway.com', 'www.tregalloway.com'],
+        })
+
+        function onRouteChangeComplete() {
+            Fathom.trackPageview()
+        }
+        // Record a pageview when route changes
+        router.events.on('routeChangeComplete', onRouteChangeComplete)
+
+        // Unassign event listener
+        return () => {
+            router.events.off('routeChangeComplete', onRouteChangeComplete)
+        }
+    }, [])
+    return (
```

Comment:
This is a bit of a nitpick, but I'm a big fan of giving code breathing room. I generally like to have a empty space before and after any hook calls such as useEffect so that something like this return statement stands out visually better.

Reply:
I understand what you mean